docs/resources/docker.md.erb

```diff
@@ -152,11 +152,11 @@ Or execute the profile directly via URL:
 
 `plugins` returns information about Docker plugins as returned by [docker plugin ls](https://docs.docker.com/engine/reference/commandline/plugin/).
 
-    describe docker_plugin('rexray/ebs') do
-      it { should exist }
-      its('id') { should_not eq '0ac30b93ad40' }
-      its('version') { should eq '0.11.1' }
-      it { should be_enabled }
+    describe docker.plugins do
+      its('names') { should include ["store/weaveworks/net-plugin", "docker4x/cloudstor"] }
```

Comment:
I'm curious, did you see 'compare using include and an array somewhere'?  Where?

Reply:
@clintoncwolfe one example here: https://www.inspec.io/docs/reference/resources/limits_conf/ 
There are multiple examples in `limits_conf`, `auditd`, `etc_hosts_allow` and `etc_hosts_deny` (did a grep on the codebase). I'm not sure if I have used such a matcher in my tests. Yes good catch, the example here is wrong. I will update.
```
inspec> describe docker.images do
inspec>   its('ids') { should include ['sha256:6c64e61818d4d683aa975e4754d6fd5c4c252b2b1ac0a5d7734017e8f6acc3be', 'sha256:86d8fd396ed6f344bfbd80cd32c884981365c77eae64d3251c97c44cd9b4a8bf'] }  
inspec> end  

Profile: inspec-shell
Version: (not specified)

  #<Inspec::Resources::DockerImageFilter:0x0000557d5548c128>
     ×  ids should include ["sha256:6c64e61818d4d683aa975e4754d6fd5c4c252b2b1ac0a5d7734017e8f6acc3be", "sha256:86d8fd396ed6f344bfbd80cd32c884981365c77eae64d3251c97c44cd9b4a8bf"]
     expected ["sha256:6c64e61818d4d683aa975e4754d6fd5c4c252b2b1ac0a5d7734017e8f6acc3be", "sha256:86d8fd396ed6f344b...4d8b3b2df83d2695d477636", "sha256:405c277e14b2422035d4d783a48c503c914dc62ba56b160e9ca862239eb46380"] to include ["sha256:6c64e61818d4d683aa975e4754d6fd5c4c252b2b1ac0a5d7734017e8f6acc3be", "sha256:86d8fd396ed6f344bfbd80cd32c884981365c77eae64d3251c97c44cd9b4a8bf"]

Test Summary: 0 successful, 1 failure, 0 skipped
inspec> 
```
I'm not sure about the other resources

Reply:
Thanks!  refs #3162